app/domain/authentication/authn_k8s/extract_container_name.rb

```diff
@@ -22,9 +21,9 @@ def call
       private
 
       def container_name
-        annotation_value("authn-k8s/#{@service_id}/#{AUTHENTICATION_CONTAINER_NAME_ANNOTATION}") ||
-          annotation_value("authn-k8s/#{AUTHENTICATION_CONTAINER_NAME_ANNOTATION}") ||
-          annotation_value("kubernetes/#{AUTHENTICATION_CONTAINER_NAME_ANNOTATION}") ||
+        annotation_value("authn-k8s/#{@service_id}/#{Restrictions::AUTHENTICATION_CONTAINER_NAME}") ||
+          annotation_value("authn-k8s/#{Restrictions::AUTHENTICATION_CONTAINER_NAME}") ||
+          annotation_value("kubernetes/#{Restrictions::AUTHENTICATION_CONTAINER_NAME}") ||
```

Comment:
Formatting suggestion for controlling line length:

```ruby
      def container_name
        name = Restrictions::AUTHENTICATION_CONTAINER_NAME
        
        annotation_value("authn-k8s/#{@service_id}/#{name}") ||
          annotation_value("authn-k8s/#{name}") ||
          annotation_value("kubernetes/#{name}") ||
          default_authentication_container_name
      end
```

Reply:
done